### PR TITLE
Cloud security posture dashboard 8.12 updates

### PR DESCRIPTION
**Pull request by benironside**
Addresses #4529 (along with twin [serverless PR](https://github.com/elastic/staging-serverless-security-docs/pull/255)) by updating the cloud posture dashboard pages (three pages due to classic docs limitations that prevent single-sourcing). Image and text updates reflect a redesigned UI that groups monitored resources by CIS benchmark rather than highlighting compliance scores for individual accounts. This makes the dashboard more scalable for larger orgs.

Previews: 
Cloud posture dashboard pages [one](https://security-docs_bk_4530.docs-preview.app.elstc.co/guide/en/security/master/cspm-posture-dashboard.html), [two](https://security-docs_bk_4530.docs-preview.app.elstc.co/guide/en/security/master/cloud-posture-dashboard.html), and [three](https://security-docs_bk_4530.docs-preview.app.elstc.co/guide/en/security/master/cloud-nat-sec-posture-dashboard.html)

### CODE REVIEW

File: docs/cloud-native-security/cspm-cloud-posture-dashboard.asciidoc [hunk 1]
```diff
@@ -24,13 +24,13 @@ The Cloud Security Posture dashboard shows:
...
+The remaining summary cards show your overall compliance score, and your compliance score for each CIS section. Click *View all failed findings* to view all failed findings, or click a CIS section name to view failed findings from only that section on the Findings page.
 
-Below the summary section, each row shows the CSP of a single cloud account or Kubernetes cluster, including its name, when its posture was last evaluated, its compliance score, and failed findings grouped by CIS section. When you begin to monitor a new account or cluster, a new row appears.
+Below the summary section, each row shows the CSP for a benchmark that applies to your monitored cloud resources. For example, if you are monitoring GCP and Azure cloud accounts, a row appears for CIS GCP and another appears for GCP Azure. Each row shows the CIS benchmark, the number of cloud accounts it applies to, its overall compliance score, and its compliance score grouped by CIS section.
```
Comment (tinnytintin10) [inline docs/cloud-native-security/cspm-cloud-posture-dashboard.asciidoc:33]: @benironside replace GCP Azure with CIS Azure

File: docs/dashboards/cloud-posture.asciidoc [hunk 2]
```diff
@@ -25,11 +25,11 @@ The Cloud Security Posture dashboard shows:
...
+The remaining summary cards show your overall compliance score, and your compliance score for each CIS section. Click *View all failed findings* to view all failed findings, or click a CIS section name to view failed findings from only that section on the Findings page.
 
-Below the summary section, each row shows the CSP of a single Cloud account or Kubernetes cluster, including its name, when its posture was last evaluated, its compliance score, and failed findings grouped by CIS section. When you begin to monitor a new account or cluster, a new row appears.
+Below the summary section, each row shows the CSP for a benchmark that applies to your monitored cloud resources. For example, if you are monitoring GCP and Azure cloud accounts, a row appears for CIS GCP and another appears for GCP Azure. Each row shows the CIS benchmark, the number of cloud accounts it applies to, its overall compliance score, and its compliance score grouped by CIS section.
```
Comment (tinnytintin10) [inline docs/dashboards/cloud-posture.asciidoc:32]: CIS Azure instead of GCP Azure

**Review by tinnytintin10 — CHANGES_REQUESTED** [review]
Requested some minor changes

File: docs/cloud-native-security/kspm-cloud-posture-dashboard.asciidoc [hunk 3]
```diff
@@ -11,8 +11,8 @@ image::images/cloud-sec-dashboard.png[The cloud Security dashboard]
 The Cloud Security Posture dashboard shows:
 
 * Configuration risk metrics for all monitored cloud accounts and Kubernetes clusters
-* Configuration risk metrics for individual cloud accounts and Kubernetes clusters
-* Configuration risks grouped by CIS Section (security guideline category)
+* Configuration risk metrics grouped by the applicable benchmark, for example CIS GCP, CIS Azure, CIS Kubernetes, or CIS EKS
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/kspm-cloud-posture-dashboard.asciidoc:14]:
```suggestion
* Configuration risk metrics grouped by the applicable benchmark, for example, CIS GCP, CIS Azure, CIS Kubernetes, or CIS EKS
```

File: docs/dashboards/cloud-posture.asciidoc [hunk 4]
```diff
@@ -10,8 +10,8 @@ image::images/cloud-sec-dashboard.png[The cloud Security dashboard]
 The Cloud Security Posture dashboard shows:
 
 * Configuration risk metrics for all monitored cloud accounts and Kubernetes clusters
-* Configuration risk metrics for individual cloud accounts and Kubernetes clusters
-* Configuration risks grouped by CIS Section (security guideline category)
+* Configuration risk metrics grouped by the applicable benchmark, for example CIS GCP, CIS Azure, CIS Kubernetes, or CIS EKS
```
Comment (nastasha-solomon) [inline docs/dashboards/cloud-posture.asciidoc:13]:
```suggestion
* Configuration risk metrics grouped by the applicable benchmark, for example, CIS GCP, CIS Azure, CIS Kubernetes, or CIS EKS
```

**Review by nastasha-solomon — DISMISSED** [review]
Looks great!

**Review by nastasha-solomon — APPROVED** [review]

**Pull request merged**